focil.md

```diff
@@ -0,0 +1,16 @@
+# FOCIL Metrics
+
+The following metrics are proposed to be added to clients for FOCIL monitoring. This list is open for discussion. Each client has the opportunity to contribute to it by suggesting additions or disputing existing metrics.
+
+| Name | Metric type | Usage | Sample collection event |
+|--------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
+| `beacon_inclusion_lists_size_bytes_total`            | Counter   | Byte size of the inclusion list  |      |
+| `beacon_inclusion_lists_validated_total`           | Counter   | Number of validated inclusion lists |      |
```

Comment:
Does this include both valid and invalid inclusion lists?

Reply:
Does the validation happen both when an IL is received over the P2P network and via RPC request?